docs/annexes/including-security-information-in-SPDX.md

```diff
@@ -0,0 +1,306 @@
+# 1. Including security information in a SPDX document
+
+SPDX 3.0 has the concept of an [__External Reference__](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Classes/ExternalRef.md) for an Element which points to "a resource outside the scope of the SPDX-3.0 content that provides additional characteristics of an Element." 
+
+The specification for External Reference types has many [type options](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Vocabularies/ExternalRefType.md), a large handful of which pertain specifically to security use cases:
+
+* secureSoftwareAttestation: A reference to information assuring that the software is developed using security practices as defined by [NIST SP 800-218 Secure Software Development Framework (SSDF)](https://csrc.nist.gov/publications/detail/sp/800-218/final) or [CISA Secure Software Development Attestation Form](https://www.cisa.gov/sites/default/files/2023-04/secure-software-self-attestation_common-form_508.pdf).
```

Comment:
NIST SP 800-218:
- Do we need specify the version number?
- "Secure Software Development Framework (SSDF) Version 1.1"

CISA Secure Software Development Attestation Form:
- The current link points to a draft version
- May need to update to the final version here: https://www.cisa.gov/resources-tools/resources/secure-software-development-attestation-form

Reply:
I just realised that the text are from `ExternalRefType.md`, so I made PR for changes in the spdx-3-model source repo here as well: https://github.com/spdx/spdx-3-model/pull/708

Reply:
Thanks for the feedback @bact -- I will update my PR with your changes.